api/global/internal/meter.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
```suggestion
// Copyright The OpenTelemetry Authors
```

But this is likely to be fixed in a rebase.